src/Umbraco.Web.UI.Client/src/installer/installer.service.js

```diff
@@ -329,20 +304,52 @@ angular.module("umbraco.install").factory('installerService', function ($rootSco
 
 		complete : function(){
 
-			service.status.progress = "100%";	
-			service.status.done = true;
-			service.status.feedback = "Redirecting you to Umbraco, please wait";
-			service.status.loading = false;
-
-			if (factTimer) {
-			    clearInterval(factTimer);
-			}
-
-			$timeout(function(){
-				window.location.href = Umbraco.Sys.ServerVariables.umbracoBaseUrl;
-			}, 1500);
+            $http.post(Umbraco.Sys.ServerVariables.installApiBaseUrl + "CompleteInstall", _installerModel)
```

Comment:
How come this is required? by the time the installer gets to this `complete` stage the site will have already shut down and then we are redirecting to the website below which will restart it

Reply:
The site will not restart in the installer (Only if the starterkit steps is applied).

This is a more explicit restart, and I also had to increase the wait time ,because IIEexpress is slower to do the restart stuff. I had an 502 error Bad request throwed by the AspNetCoreModule, outside of our application.

Reply:
Cool all good then